adagrams/game.py

```diff
@@ -1,13 +1,129 @@
 from random import randint
 
 def draw_letters():
-    pass
+    hand_length = 10
+    draw_pool = build_draw_pool()
+    hand = []
+    for i in range(hand_length):
+        random_num = randint( 0, len(draw_pool) - 1)
+        hand.append(draw_pool.pop(random_num))
+    return(hand)
 
 def uses_available_letters(word, letter_bank):
-    pass
+    uppercase_word = word.upper()
+    if not set(uppercase_word).issubset(letter_bank):
+        return False
+    copy_letter_bank = letter_bank.copy()
+    for letter in uppercase_word:
+        if letter in copy_letter_bank:
+            copy_letter_bank.remove(letter)
+        else:
+            return False
+    return True
 
 def score_word(word):
-    pass
+    if not word:
+        return 0
+    score_total = 0
+    if len(word) > 6:
+        score_total = 8
```

Comment:
Consider adding a few blank lines within your function to separate logically distinct sections of the code (e.g., setup, processing, return).  According to [PEP 8](https://peps.python.org/pep-0008/#blank-lines), blank lines can improve readability by visually grouping related operations. Right now, a lot of your functions' logic run together, which can make it harder to quickly understand the structure and flow of your function.
```suggestion
    if not word:
        return 0
        
    score_total = 0
    if len(word) > 6:
        score_total = 8
```